### PR TITLE
Replace default tests with rspec

### PR DESCRIPTION
**Pull request by alexeyschepin**
Install rspec and make basic setup.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,8 @@ public/packs-test
 spec/examples.txt
 yarn-error.log
 storage
+
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
```
Comment (manuelmeurer) [inline .gitignore:12]: Please remove these changes.

File: Gemfile [hunk 2]
```diff
@@ -31,3 +31,8 @@ group :development do
   gem 'spring',                                 '~> 2.0'
   gem 'spring-watcher-listen',                  '~> 2.0'
 end
+
+group :development, :test do
+  gem 'factory_bot_rails'
+  gem 'rspec-rails', '~> 4.0.1'
+end
```
Comment (manuelmeurer) [inline Gemfile:38]: Please add similar version constraints as the other gems have, and indent the constraint properly.

**Review by manuelmeurer — CHANGES_REQUESTED** [review]
Thanks for your effort, @alexeyschepin!
Could you address the two comments please?

**Comment by alexeyschepin** [comment]
Hey @manuelmeurer, have a look at the fix, please.

**Comment by manuelmeurer** [comment]
Thanks, @alexeyschepin and sorry for the delay... 🙇‍♂️

**Pull request merged**